Publish Failed

## What happened?
https://github.com/pomerium/datasource/actions/runs/3128409488/jobs/5076210611

```
#23 [linux/amd64 build 8/8] RUN make build
#23 0.143 fatal: not a git repository (or any of the parent directories): .git
#23 0.145 fatal: not a git repository (or any of the parent directories): .git
#23 0.147 ==> build
#23 1.285 cmd/pomerium-datasource/directory.go:9:2: no required module provides package github.com/pomerium/datasource/pkg/directory; to add it:
#23 1.285 	go get github.com/pomerium/datasource/pkg/directory
#23 1.288 cmd/pomerium-datasource/directory.go:10:2: no required module provides package github.com/pomerium/datasource/pkg/directory/auth0; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/auth0
#23 1.288 cmd/pomerium-datasource/directory.go:11:2: no required module provides package github.com/pomerium/datasource/pkg/directory/azure; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/azure
#23 1.288 cmd/pomerium-datasource/directory.go:12:2: no required module provides package github.com/pomerium/datasource/pkg/directory/github; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/github
#23 1.288 cmd/pomerium-datasource/directory.go:13:2: no required module provides package github.com/pomerium/datasource/pkg/directory/gitlab; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/gitlab
#23 1.288 cmd/pomerium-datasource/directory.go:14:2: no required module provides package github.com/pomerium/datasource/pkg/directory/google; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/google
#23 1.288 cmd/pomerium-datasource/directory.go:15:2: no required module provides package github.com/pomerium/datasource/pkg/directory/okta; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/okta
#23 1.288 cmd/pomerium-datasource/directory.go:16:2: no required module provides package github.com/pomerium/datasource/pkg/directory/onelogin; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/onelogin
#23 1.288 cmd/pomerium-datasource/directory.go:17:2: no required module provides package github.com/pomerium/datasource/pkg/directory/ping; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/ping
#23 1.290 make: *** [Makefile:45: build] Error 1
#23 ERROR: process "/bin/sh -c make build" did not complete successfully: exit code: 2

#20 [linux/arm64 build 5/8] RUN go mod download
#20 CANCELED
------
 > [linux/amd64 build 8/8] RUN make build:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/gitlab
#23 1.288 cmd/pomerium-datasource/directory.go:14:2: no required module provides package github.com/pomerium/datasource/pkg/directory/google; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/google
#23 1.288 cmd/pomerium-datasource/directory.go:15:2: no required module provides package github.com/pomerium/datasource/pkg/directory/okta; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/okta
#23 1.288 cmd/pomerium-datasource/directory.go:16:2: no required module provides package github.com/pomerium/datasource/pkg/directory/onelogin; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/onelogin
#23 1.288 cmd/pomerium-datasource/directory.go:17:2: no required module provides package github.com/pomerium/datasource/pkg/directory/ping; to add it:
#23 1.288 	go get github.com/pomerium/datasource/pkg/directory/ping
#23 1.290 make: *** [Makefile:45: build] Error 1
------
pomerium-datasource.dockerfile:12
--------------------
  10 |     COPY ./cmd/ ./cmd/
  11 |     COPY ./internal/ ./internal/
  12 | >>> RUN make build
  13 |     
  14 |     FROM gcr.io/distroless/base-debian11@sha256:a08c76433d484[340](https://github.com/pomerium/datasource/actions/runs/3128409488/jobs/5076210749#step:7:343)bd97013b5d868edfba797fbf83dc82174ebd0768d12f491d
--------------------
ERROR: failed to solve: process "/bin/sh -c make build" did not complete successfully: exit code: 2
Error: buildx failed with: ERROR: failed to solve: process "/bin/sh -c make build" did not complete
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -1,0 +1,29 @@
+name: Build
+permissions:
+  contents: read
+on:
+  pull_request: {}
+
+jobs:
+  dockerhub:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          fetch-depth: 0
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@8b122486cedac8393e77aa9734c3528886e4a1a8
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@dc7b9719a96d48369863986a06765841d7ea23f6
+
+      - name: Build
+        uses: docker/build-push-action@c84f38281176d4c9cdb1626ffafcd6b3911b5d94
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          file: pomerium-datasource.dockerfile
+          cache-from: type=gha
+          cache-to: type=gha,mode=max

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -5,7 +5,7 @@ on:
   pull_request: {}
 
 jobs:
-  dockerhub:
+  docker:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout

--- a/pomerium-datasource-ip2location.dockerfile
+++ b/pomerium-datasource-ip2location.dockerfile
@@ -20,6 +20,7 @@ RUN go mod download
 
 COPY ./cmd/ ./cmd/
 COPY ./internal/ ./internal/
+COPY ./pkg/ ./pkg/
 RUN make build
 
 FROM gcr.io/distroless/base-debian11@sha256:a08c76433d484340bd97013b5d868edfba797fbf83dc82174ebd0768d12f491d

--- a/pomerium-datasource.dockerfile
+++ b/pomerium-datasource.dockerfile
@@ -9,6 +9,7 @@ RUN go mod download
 
 COPY ./cmd/ ./cmd/
 COPY ./internal/ ./internal/
+COPY ./pkg/ ./pkg/
 RUN make build
 
 FROM gcr.io/distroless/base-debian11@sha256:a08c76433d484340bd97013b5d868edfba797fbf83dc82174ebd0768d12f491d

